_includes/events-hero.html

```diff
@@ -0,0 +1,40 @@
+{% macro render(hero) %}
+
+{% from "post-macros.html" import category_icon as category_icon %}
+
+<section class="content_hero hero">
+    <div class="hero_wrapper">
+        <a href="#" class="hero_card hero_card__events">
```

Comment:
Since the hero card is a distinct block of content, I'd suggest wrapping it in a `<section>`. 

Here's a rough snippet with semantically appropriate elements:

```html
<section class="hero_card">
      <a href="#" class="hero_card__events">
            <div class="hero_card-bg">
               <div class="hero_card-padding">
                    <header class="category-slug">
                        <span class="category-slug_icon cf-icon cf-icon-bullhorn"></span>
                        <span class="u-visually-hidden">Category:</span>
                        Event Category Name
                        <h1 class="summary_header">
                            Event Title Here
                        </h1>
                    </header>
                    <p class="summary_text">
                            Include a short summary to give users a quick overview of the event.
                    </p>
                    <footer>
                        <time class="summary_event-date" datetime="2015-01-21T2:30">1/21/2015 @ 2:30 PM CST</time>
                    </footer>
                </div>
            </div>
        </a>
</section>
```

**Caveat:** This may be breaking existing patterns I don't know about and there may not be a polyfill already in place for `<time>`.

Also, why does `hero_card-padding` include a margin? That should probably be noted in an issue.